python/social_bot/envs/play_ground.py

```diff
@@ -93,6 +95,7 @@ def __init__(self,
             world_name (string): Select the world file, e.g., empty.world, play_ground.world,
                 grocery_ground.world
             tasks (list): a list of teacher.Task, e.g., GoalTask, KickingBallTask
+            goal_conditioned (bool): Turn on goal conditioned tasks.
```

Comment:
comment what it the difference between goal conditioned task and non-goal-conditioned tasks, especially the observation.

Reply:
Done.  PTAL @emailweixu 

Thanks,
Le